creator-node/sequelize/migrations/20220427014326-add_playlist_to_clockrecords_enum.js

```diff
@@ -0,0 +1,129 @@
+'use strict'
+
+/**
+ * New table - Playlist
+ * CNodeUsers Table considered a Reference Table only
+ */
+
+const tableName = 'Playlists'
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    console.log('STARTING MIGRATION 20220427014326-add_playlist_to_clockrecords_enum')
+    const transaction = await queryInterface.sequelize.transaction()
+    
+    // Create Playlist table
+    await createPlaylistTable(queryInterface, Sequelize, transaction)
+
+    await addPlaylistToClockRecordSourceTables(queryInterface, Sequelize, transaction)
+    
+    // Add composite unique constraint on (playlistId, clock) to Playlist
+    await addCompositeUniqueConstraints(queryInterface, Sequelize, transaction)
+
+    await transaction.commit()
+    console.log('FINISHED MIGRATION add_playlist_to_clockrecords_enum')
+  },
+
+  down: async (queryInterface, Sequelize) => {
+    await removeCompositeUniqueConstraints(queryInterface, Sequelize)
```

Comment:
same thing here - this call is redundant with dropPlaylistTable
also i dont think it works since it doesn't pass in the table name :)

Reply:
thanks for adding the down migration - would you mind also testing it? i know kind of annoying but if we're gonna have this code should make sure it works